src/python/pcmdi/bias_xyt.py

```diff
@@ -0,0 +1,14 @@
+import MV2
+import cdutil
+
+def compute(dm, do):
+    """ Computes bias"""
+    if dm is None and do is None:  # just want the doc
+        return {
+            "Name": "Bias",
+            "Abstract": "Compute Full Average of Model - Observation",
+            "Contact": "Peter Gleckler <gleckler1@llnl.gov>",
```

Comment:
@doutriaux1 @gleckler1 it might be worth doing a global replace throughout the whole repo?